qemu/tests/cfg/hugepage_specify_node.cfg

```diff
@@ -0,0 +1,9 @@
+- hugepage_specify_node:
+    type = hugepage_specify_node
+    virt_test_type = qemu
+    kill_vm = yes
+    start_vm = no
+    not_preprocess = yes
+    mem = 4096
+    idle_node_mem = 1024
+    extra_params += " -mem-path /mnt/kvm_hugepage"
```

Comment:
Seems the path is not mounted. Please double check.

Reply:
checking

Reply:
correct